tests/integration/targets/mso_schema_template_migrate/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+    host: '{{ mso_hostname }}'
+    username: '{{ mso_username }}'
+    password: '{{ mso_password }}'
+    validate_certs: '{{ mso_validate_certs | default(false) }}'
+    use_ssl: '{{ mso_use_ssl | default(true) }}'
+    use_proxy: '{{ mso_use_proxy | default(true) }}'
+    output_level: '{{ mso_output_level | default("info") }}'
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    vrf: VRF
+    layer3_multicast: true
+    state: present
+
+- name: Ensure ANP exist
+  mso_schema_template_anp:
+    host: '{{ mso_hostname }}'
+    username: '{{ mso_username }}'
+    password: '{{ mso_password }}'
+    validate_certs: '{{ mso_validate_certs | default(false) }}'
+    use_ssl: '{{ mso_use_ssl | default(true) }}'
+    use_proxy: '{{ mso_use_proxy | default(true) }}'
+    output_level: '{{ mso_output_level | default("info") }}'
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    anp: ANP
+    state: present
+
+- name: Ensure ANP2 exist
+  mso_schema_template_anp:
+    host: '{{ mso_hostname }}'
+    username: '{{ mso_username }}'
+    password: '{{ mso_password }}'
+    validate_certs: '{{ mso_validate_certs | default(false) }}'
+    use_ssl: '{{ mso_use_ssl | default(true) }}'
+    use_proxy: '{{ mso_use_proxy | default(true) }}'
+    output_level: '{{ mso_output_level | default("info") }}'
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    anp: ANP2
+    state: present
+
+- name: Ensure ansible_test_1 BD exist
+  mso_schema_template_bd:
+    <<: *vrf_present
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    bd: ansible_test_1
+    vrf:
+      name: VRF
+    layer3_multicast: true
+    state: present
+
+- name: Ensure EPG exist
+  mso_schema_template_anp_epg: &epg_present
+    host: '{{ mso_hostname }}'
+    username: '{{ mso_username }}'
+    password: '{{ mso_password }}'
+    validate_certs: '{{ mso_validate_certs | default(false) }}'
+    use_ssl: '{{ mso_use_ssl | default(true) }}'
+    use_proxy: '{{ mso_use_proxy | default(true) }}'
+    output_level: '{{ mso_output_level | default("info") }}'
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    anp: ANP
+    epg: ansible_test_1
+    bd:
+      name: ansible_test_1
+    vrf:
+      name: VRF
+    state: present
+  register: cm_add_epg
+
+- name: Add EPG 2 (normal mode)
+  mso_schema_template_anp_epg:
+    <<: *epg_present
+    anp: ANP2
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    epg: ansible_test_2
+
+- name: Migration of objects between templates
+  mso_schema_template_migrate:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template_1
+    target_schema: '{{ mso_schema | default("ansible_test_2") }}'
+    target_template: Template_1
+    bds:
+    - ansible_test_1
+    epgs:
+    - epg: ansible_test_1
+      anp: ANP
```

Comment:
Could it be possible that we only migrate epgs or bds?
if it's possible, can we test it?

Reply:
yes, made changes and tested.